### ISSUE TITLE
Should ufw_outgoing_traffic also allow to specify the protocol?

### ISSUE THREAD

**Issue by jdaln** [konstruktoid/ansible-role-hardening#803]
Hello!

For ufw_outgoing_traffic, should it allow to specify the protocol? For example, we would have:
```
ufw_outgoing_traffic:
  - 22/tcp
  - 53 # this means both, as is currently the case
  - 80/tcp
etc...
```
I can make the PR with the necessary edits to:
https://github.com/konstruktoid/ansible-role-hardening/blob/9045255668abd6c04284926aa515f34659a414cf/tasks/ufw.yml#L130-L137

**Comment by konstruktoid** [comment]
yeah, that seems like a good addition.

something line:
```yaml
  vars:
    ufw_outgoing_traffic:
      - { port: 80, proto: 'tcp' }
      - 443
      - 53
      - 123
      - 22
      - { port: 1234, proto: 'udp' }
  tasks:
    - name: Allow outgoing specified ports
      become: true
      community.general.ufw:
        rule: allow
        to_port: "{{ item.port | default(item) }}"
        proto: "{{ item.proto | default('tcp') }}"
        direction: out
        comment: ansible managed
      loop: "{{ ufw_outgoing_traffic }}"
```

**Comment by konstruktoid** [comment]
closed by #812
